src/borg/testsuite/upgrader.py

```diff
@@ -149,8 +149,8 @@ def test_convert_all(attic_repo, attic_key_file, inplace):
     """
     repo_path = attic_repo
 
-    # check should fail because of magic number
-    assert not repo_valid(repo_path)
+    with pytest.raises(Repository.AtticRepository):
+        repo_valid(repo_path)
```

Comment:
Nowhere. It is defined in `src/borg/testsuite/upgrader.py` and does not have any other call sites.